### PR TITLE
python3packages.meteoalertapi: init at 0.1.8

### PR DESCRIPTION
**Pull request by fabaff**
###### Motivation for this change
Python wrapper for MeteoAlarm.org

https://github.com/rolfberkenbosch/meteoalert-api

This is a Home Assistant dependency.

- Related Home Assistant change: https://github.com/home-assistant/core/pull/51383
- Target Home Assistant release: current (only dependency update, no code changes)

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Added a release notes entry if the change is major or breaking
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/python-modules/meteoalertapi/default.nix [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    xmltodict
+  ];
+
+  # Tests require network access
+  doCheck = false;
+
+  pythonImportsCheck = [ "meteoalertapi" ];
+
+  meta = with lib; {
+    description = " Python wrapper for MeteoAlarm.org";
```
Comment (legendofmiracles) [inline pkgs/development/python-modules/meteoalertapi/default.nix:32]:
```suggestion
    description = "Python wrapper for MeteoAlarm.org";
```

File: pkgs/development/python-modules/meteoalertapi/default.nix [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    homepage = "https://github.com/rolfberkenbosch/meteoalert-api";
+    license = with licenses; [ cc0 ];
```
Comment (legendofmiracles) [inline pkgs/development/python-modules/meteoalertapi/default.nix:34]:
```suggestion
    license = with licenses; [ mit ];
```
readme and LICENSE say it's licensed under the mit

File: pkgs/development/python-modules/meteoalertapi/default.nix (hunk 1 above)
Comment (fabaff) [reply pkgs/development/python-modules/meteoalertapi/default.nix]: Thanks

File: pkgs/development/python-modules/meteoalertapi/default.nix (hunk 2 above)
Comment (fabaff) [reply pkgs/development/python-modules/meteoalertapi/default.nix]: Thanks, fixed.

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 125559` at 10458d7a run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages built successfully:</summary>
  <ul>
    <li>python38Packages.meteoalertapi</li>
    <li>python39Packages.meteoalertapi</li>
  </ul>
</details>

**Review by jonringer — APPROVED** [review]
LGTM

Result of nixpkgs-review pr 125559 run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)<details>  <summary>2 packages built:</summary>  <ul>    <li>python38Packages.meteoalertapi</li>    <li>python39Packages.meteoalertapi</li>  </ul></details>

**Pull request merged**